src/main/java/com/deviceinsight/kafka/health/cache/CaffeineCacheServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package com.deviceinsight.kafka.health.cache;
+
+import com.github.benmanes.caffeine.cache.Cache;
+import com.github.benmanes.caffeine.cache.Caffeine;
+
+import java.util.concurrent.TimeUnit;
+
+public class CaffeineCacheServiceImpl implements CacheService<String> {
+
+	private final Cache<String, String> cache;
+
+	public CaffeineCacheServiceImpl(long expireAfterWrite) {
+		this.cache = Caffeine.newBuilder()
```

Comment:
I think you should use the Caffeine `Cache` directly instead of declaring your own interface and implementation (`CacheService`, `CaffeineCacheServiceImpl`).